Makefile

```diff
@@ -159,3 +160,56 @@ sv: clean_tmp mkdirs copy_assets build_sv finalize
 tr: clean_tmp mkdirs copy_assets build_tr finalize
 zh-CN: clean_tmp mkdirs copy_assets build_zh-CN finalize
 zh-TW: clean_tmp mkdirs copy_assets build_zh-TW finalize
+
+# Localizations
+
+localize_ar:
+	$(LIVESCRIPT_BIN) $(LOCALIZE_SCRIPT) ar
```

Comment:
Ouch, I'd never seen the whole Makefile, only this diff. Why don't you let me take a poke at it, there are actually several things that could be fixed.